activities/trainings/github-for-newcomers.md

```diff
@@ -0,0 +1,29 @@
+# GitHub for newcomers
+
+This guide explains: How to learn to use GitHub effectively for working with the Foundation for Public Code projects.
+
+GitHub is an incredibly powerful tool for collaboration at scale. However it is a power tool and requires some serious learning and understanding. For this reason, in order to learn effectively and not be frustrated often it pays to really dive deep into what it is and how it works.
+
+If you know what a rebase is, this guide is not for you. If you don't, we expect you to know and understand everything that is in these courses.
```

Comment:
I've done everything in this course, and still don't know what a rebase is :(

Reply:
Yeah, I've used that more as a high-water marker since I didn't really know anything simpler than that that might be too easy to know but these courses might still be good for you.

If you are interested: <https://www.atlassian.com/git/tutorials/merging-vs-rebasing>

Reply:
I also like the github guides youtube videos for context: 
https://www.youtube.com/githubguides
should we add these?

Reply:
Yes @nebster, quite off-topic for this conversation, however I've added it.

@ElenaFdR is this resolved or should we remove that mention?